src/app/utils/MGPFallible.ts

```diff
@@ -9,6 +9,9 @@ export abstract class MGPFallible<T> {
         return new MGPFallibleFailure(reason);
     }
 
+    protected constructor() {
```

Comment:
If you feel like removing uneeded breakline !

Reply:
celle d'après le constructeur ? done